framework/mcp/requirements.txt

```diff
@@ -1,2 +1,5 @@
 cratedb-toolkit
-mcp<1.5
+mcp<1.6
+mcp-alchemy @ git+https://github.com/runekaagaard/mcp-alchemy.git@b85aae6; python_version>='3.12'
+sqlalchemy-cratedb>=0.42.0.dev1
+where
```

Comment:
Python version constraint with mcp-alchemy can be removed after merging this one.

- https://github.com/runekaagaard/mcp-alchemy/pull/11